app/views/admin/barcode_items/_barcode_modal.html.erb

```diff
@@ -1,33 +1,38 @@
 <div id="newBarcode" class="modal fade">
-  <div class="modal-dialog">
+  <div class="modal-dialog modal-lg">
```

Comment:
Did this change somehow come from the ruby upgrade?

Reply:
Nope.  We recently updated the theme and I noticed there barcode modal was a little messed up and it felt like overkill adding a separate PR for some small cosmetic changes.

Reply:
Cool, as long as it is on purpose! :)